include/sway/config.h

```diff
@@ -273,6 +267,7 @@ enum render_bit_depth {
 struct output_config {
 	char *name;
 	int enabled;
+	int power;
```

Comment:
Why not make `power` a `enum zwlr_output_power_v1_mode`?

Reply:
1. I don't want to tie this to a particular Wayland protocol
2. Consistency with the rest of the fields
3. Doesn't have a "default"/"unset" enum entry